package.json

```diff
@@ -238,9 +238,7 @@
                     "type": "array",
                     "default": [
                         "Ninja",
-                        "Unix Makefiles",
-                        "MinGW Makefiles",
-                        "NMake Makefiles"
+                        "Unix Makefiles"
```

Comment:
I removed NMake and MinGW generators because they should be provided by Environment provider.
However it seems NMake is preferred over msbuild by some.
Benefit of MSBuild is that it works even without activating environment or from plain command line.